### PR TITLE
NEPT-2380: Add prefix to purged alias if needed.

### PR DESCRIPTION
**Pull request by drishu**
### Description

If language negotiation selection is configured with url prefix, add this prefix to purged alias.

### Change log

- Added: Prefix in purge alias.

### CODE REVIEW

File: profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module [hunk 1]
```diff
@@ -546,63 +546,91 @@ function _nexteuropa_varnish_get_node_paths($node) {
...
-    // Add escaped alias.
-    // Example content\/my\-page.
-    $paths['escaped_alias'] = preg_quote($original_alias, '/');
   }
+
   return $paths;
 }
 
+/**
+ * Custom callback to escape all path at the end of the collection.
```
Comment (drishu) [inline profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module:717]: Please add parameter and return value description in function comment.

File: profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module [hunk 2]
```diff
@@ -546,63 +546,91 @@ function _nexteuropa_varnish_get_node_paths($node) {
...
+ */
+function _nexteuropa_varnish_escape($path) {
+  return preg_quote($path, '/');
+}
+
+/**
+ * Custom function returns alias depending of language & translation mechanism.
+ */
```
Comment (drishu) [inline profiles/common/modules/custom/nexteuropa_varnish/nexteuropa_varnish.module:745]: Please add parameter and return value description in function comment.

**Review by drishu — COMMENTED** [review]
Just small standards thing.

**Review by Fefaine — APPROVED** [review]
Approving after Andras and I both reviewed our mutual changes

**Review by AlbaParrilla — APPROVED** [review]

**Pull request merged**